.misc/deploy.pypi.sh

```diff
@@ -2,22 +2,40 @@
 
 # Do not use `set -x` here as then it displays the PYPIPW in logs
 set -e
-
 # Get environment variables, readily decrypted by rultor
 source ../rultor_secrets.sh
```

Comment:
Why is this line running in non-rultor mode.
It obviously was not tested. Please label your PR 'wip' until it is working (or test on your own travis on your own fork on a different branch name).